### PR TITLE
Fix lint warnings/errors

### LINKED ISSUE #98: Fix lint warnings
We have something like 400 lint warnings. We need to go through them and either fix or suppress.

### PR DESCRIPTION
**Pull request by zoranzoki21**
Fixes #98 

Changes: Fixed >100 lint warnings/errors found by Android Studio

### CODE REVIEW

File: app/src/main/res/values-af/strings.xml [hunk 1]
```diff
@@ -10,8 +10,7 @@
   <string name="menu_fullscreen">Volle skerm</string>
```
Comment (mhutti1) [inline app/src/main/res/values-af/strings.xml]: Strings are auto generated so don't commit these changes.

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -17,7 +17,6 @@
   <string name="save_media_error">An error occurred when trying to save the media!</string>
   <string name="save_media_saved">Saved media as %s to Android/media/org.kiwix…/</string>
```
Comment (mhutti1) [inline app/src/main/res/values/strings.xml]: This is the only strings file that should be edited.

Comment (zoranzoki21) [reply app/src/main/res/values/strings.xml]: Ok. Thanks! I no know why this build failing..

File: app/src/main/res/values-af/strings.xml (hunk 1 above)
Comment (zoranzoki21) [reply app/src/main/res/values-af/strings.xml]: Ok. I will not commit these changes. Can you tell me where this app can be translated?

**Comment by zoranzoki21** [comment]
I no know why this failing.

File: app/build.gradle [hunk 3]
```diff
@@ -72,14 +73,14 @@ dependencies {
...
-  androidTestImplementation 'com.android.support.test:runner:1.0.1'
-  androidTestImplementation 'com.android.support.test:rules:1.0.1'
+  androidTestImplementation 'com.android.support.test:runner:1.0.2'
+  androidTestImplementation 'com.android.support.test:rules:1.0.2'
 
   // Tab indicator
   implementation "com.pacioianu.david:ink-page-indicator:$inkPageIndicatorVersion"
 
   // Guava
-  implementation group: 'com.google.guava', name: 'guava', version: '21.0'
```
Comment (abdulwd) [inline app/build.gradle]: Revert the changes made in this gradle file the build will pass. Use 2 space tabs everywhere instead of 4.

**Comment by abdulwd** [comment]
It seems you have removed some imports which are needed. First, run **Build -> Clean Project** then try to build the project from **Build -> Rebuild project** and install the app. If you're able to install the app without any problems then run lint from **Analyze -> Inspect Code**. Open a file with lint warning and fix all the lint warning in it. Limit your fixes to at most 40 warning because greater than that will make the PR difficult to review.
If you face any problem ping me. :)

**Comment by zoranzoki21** [comment]
I will make new pull request.
